vxyowsup/whatsapp.py

```diff
@@ -107,9 +115,19 @@ def _kill():
         self.stack.execDetached(_stop)
         self.stack.execDetached(_kill)
 
+    def send_to_stack(self, text, toAddress):
+        # encapsulate text and address?
+        # rather have it interact with stack only?
+        self.stack.execDetached(self.echo_layer.send, text, toAddress)
+
 
 class EchoLayer(YowInterfaceLayer):
 
+    def send_to_human(self, text, toAddress):
+        message = TextMessageProtocolEntity(text, to=toAddress)
+        self.toLower(message)
+        print(text)
```

Comment:
Perhaps we should print the address here too in case it's important for debugging?